src/main/resources/application.properties

```diff
@@ -8,7 +8,7 @@ kafka.producer.acks=1
 kafka.consumer.auto.offset.reset=earliest
 
 #AMQP related settings
-amqp.enabled=true
+#amqp.enabled=true
```

Comment:
Does this change really belong in this PR?

Reply:
No you are right, I was testing offline with postman and removed the AMQP stuff.